### PR TITLE
Translations for ja.yml through line 175

### PR DESCRIPTION
**Pull request by matcha440**
# Description 
I think the first 175 lines here were for the most part fairly straightforward, but I'm sure there are some mistakes.  I put my wife's GitHub handle as co-author as she helped a little with some of these.

## Corresponding Issue

Relates to #1215 

# Test Coverage

✅

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -27,152 +27,152 @@ ja:
...
+    faq: よくある質問
+    resources: リーソス
+    code_of_conduct: 行動規範
   common:
-    name: Name
-    you: You
-    title: Title
-    date: Date
-    daily_reminder: 'Daily reminder email'
+    name: 名前
```
Comment (matcha440) [inline config/locales/ja.yml]: I noticed when I was previewing this locally that the usage isn't quite what I thought it was.  It's not the name of a person, so 名前 may not be suitable, but I don't know what would fit better here.

**Comment by julianguyen** [comment]
Could you rebase from master? There are some new translations added related to the report feature. Sorry about that!

**Comment by matcha440** [comment]
@julianguyen yeah, I could rebase but I think that would be somewhat disruptive to the flow.  Would it be better if I just merge master into ja-translations (and fix conflicts) and from ja-translations to the branch related to this review?  I'd personally prefer to do any rebasing once we're near the end and I can just tidy up all commits prior to merging into master at that time.

**Comment by julianguyen** [comment]
@matcha440 That works! :)

File: config/locales/ja.yml [hunk 2]
```diff
@@ -27,152 +27,152 @@ ja:
...
-      premade1_name: Family
-      premade1_description: 'Family related'
-      premade2_name: Friends
-      premade2_description: 'Friends related'
-      premade3_name: Work
-      premade3_description: 'Work related'
-      premade4_name: Meditation
-      premade4_description: 'Meditation, deep breathing, and relaxing activities'
-      instructions: 'You haven''t created any custom categories yet.<br>You can also add the following premade categories and customize them another time.'
+      subtitle: '人生は大変なので、人や場所や物をカテゴライズしましょう。'
```
Comment (nshki) [inline config/locales/ja.yml]: Hmm, 人生は大変なので is a good translation here, but the whole phrase doesn't have the same ring in Japanese as it does in English. Perhaps just omitting that portion would work better for Japanese.

File: config/locales/ja.yml [hunk 3]
```diff
@@ -27,152 +27,152 @@ ja:
     fr: Français
     ja: 日本語
   navigation:
-    about: About
-    blog: Blog
-    contribute: Contribute
-    partners: Partners
-    privacy: 'Privacy Policy'
+    about: 'プロフィール'
```
[the remark this answers is not in the document]
Comment (matcha440) [reply config/locales/ja.yml]: Yeah, and that's what my wife suggested at first too, and then I saw this string is actually referenced in two different places.  One is of course the main navigation bar so "if meにつて" does make sense there but the other is in the section for modifying the Account.  It's about like we have to have two entries to handle the different contexts here but that's going to be a bit more effort.  If we don't have something that makes sense for both then that's what we may have to end up doing though, right?

**Comment by matcha440** [comment]
@nshki thank you for the initial feedback.  I'll get these updated a little later today but had a question on one of them.

File: config/locales/ja.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (nshki) [reply config/locales/ja.yml]: Ah gotcha. Yeah, unfortunately if we can't find something that'll work in both contexts, we'll have to split this.

[the remark this answers is not in the document]
Comment (matcha440) [reply config/locales/ja.yml]: I've put it back and I think it may make sense to open up a separate issue to address splitting it.

**Comment by matcha440** [comment]
Merging things in from master added some new entries, but I'll save those for later to keep the scope of this PR focused.

File: config/locales/ja.yml [hunk 4]
```diff
@@ -27,55 +27,55 @@ ja:
...
+      confirm: 'よろしいですか？'
+      delete: 消す
+      edit: 編集する
+      edit_instance: '編集%{instance}'
+      join: 参加する
+      leave: 立ち去る
+      add: 追加する
+      add_all: 全て追加する
+      select_all: 全て選択する
+      remove: 外す
```
Comment (nshki) [inline config/locales/ja.yml]: 消す I think is more commonly used, but I might be wrong.

File: config/locales/ja.yml [hunk 5]
```diff
@@ -93,103 +93,103 @@ ja:
...
-      premade2_name: Friends
-      premade2_description: 'Friends related'
-      premade3_name: Work
-      premade3_description: 'Work related'
-      premade4_name: Meditation
-      premade4_description: 'Meditation, deep breathing, and relaxing activities'
-      instructions: 'You haven''t created any custom categories yet.<br>You can also add the following premade categories and customize them another time.'
+      subtitle: '人や場所や物をカテゴライズしましょう。'
+      premade1_name: 家族
+      premade1_description: 家族に関係ある
```
Comment (nshki) [inline config/locales/ja.yml]: 家族関係, perhaps?

File: config/locales/ja.yml [hunk 6]
```diff
@@ -93,103 +93,103 @@ ja:
...
+      premade2_name: 友達
+      premade2_description: 友達に関係ある
```
Comment (nshki) [inline config/locales/ja.yml]: 友達関係

File: config/locales/ja.yml [hunk 7]
```diff
@@ -93,103 +93,103 @@ ja:
...
+      premade3_name: 仕事
+      premade3_description: 仕事に関係ある
```
Comment (nshki) [inline config/locales/ja.yml]: 仕事関係

File: config/locales/ja.yml [hunk 8]
```diff
@@ -93,103 +93,103 @@ ja:
...
+      premade4_name: 瞑想
+      premade4_description: '瞑想と深呼吸とリラックスのこと'
```
Comment (nshki) [inline config/locales/ja.yml]: 瞑想、深呼吸、とリラックス関係

File: config/locales/ja.yml [hunk 9]
```diff
@@ -93,103 +93,103 @@ ja:
...
+    singular: グループ
+    plural: グループ
     form:
-      leaders: Leaders
-      only_leader: 'You are the only leader, therefore you can''t leave.'
-      cannot_delete: 'Please note that you cannot delete groups.'
-      error_save: 'We can''t save this group because: '
-      error_edit_permission: 'You must be a leader of a group in order to edit it'
+      leaders: リーダー
+      only_leader: 'リーダーはあなたしかないから、立ち去ることができません。'
```
Comment (nshki) [inline config/locales/ja.yml]: リーダーはあなたしかいませんので、立ち去ることができません。

File: config/locales/ja.yml [hunk 10]
```diff
@@ -93,103 +93,103 @@ ja:
...
+      cannot_delete: 'グループを削除することができません。'
+      error_save: 'グループを保存できない理由は：'
+      error_edit_permission: 'グループのリーダーじゃないから編集できません。'
```
Comment (nshki) [inline config/locales/ja.yml]: あなたはグループリーダーではありませんので編集できません。

File: config/locales/ja.yml [hunk 11]
```diff
@@ -93,103 +93,103 @@ ja:
...
-      group_not_exists: 'You cannot join a group that doesn''t exist.'
+      success: 'このグループに入りました。'
+      group_not_exists: '存在しないグループが入れません。'
     leave:
-      error: 'You cannot leave the group, you are the only leader.'
-      success: 'You have left %{group}'
-      remove_member_success: 'You have removed %{user} from %{group}'
-      remove_member_error: 'You cannot kick a group member because you are not a leader.'
-    new: 'New Group'
+      error: 'グループのリーダーはあなただけから、立ち去ることができません。'
```
Comment (nshki) [inline config/locales/ja.yml]: グループリーダーはあなたしかいませんので、立ち去ることができません。

File: config/locales/ja.yml [hunk 12]
```diff
@@ -93,103 +93,103 @@ ja:
...
+      success: '%{group}を立ち去りました。'
+      remove_member_success: '%{user}が%{group}から外しました。'
+      remove_member_error: 'リーダーではないからメンバーが外せません。'
```
Comment (nshki) [inline config/locales/ja.yml]: あなたはリーダーではありませんので、メンバーを外せません。

**Pull request merged**